lib/analyzer/image-inspector.ts

```diff
@@ -59,6 +59,10 @@ async function pullWithDockerBinary(
     await docker.save(targetImage, saveLocation);
   } catch (err) {
     debug(`couldn't pull ${targetImage} using docker binary: ${err}`);
+
+    if (err.stderr.includes("unknown operating system or architecture")) {
```

Comment:
stderr is always present in the `err` object?

Reply:
Ah good catch, thank you! This is the output coming back from `subProcess.execute` (so `docker.pullCli` in this case) but from `docker.save` we'll be getting a standard error. I've fixed that :-)